libraries/AP_BattMonitor/AP_BattMonitor_FuelLevel_Analog.cpp

```diff
@@ -55,36 +55,36 @@ const AP_Param::GroupInfo AP_BattMonitor_FuelLevel_Analog::var_info[] = {
     // @Param: FL_PIN
     // @DisplayName: Fuel level analog pin number
     // @Description: Analog input pin that fuel level sensor is connected to. Airspeed ports can be used for Analog input. When using analog pin 103, the maximum value of the input in 3.3V.
-    // @Values: -1:Not Used,11:Pixracer,13:Pixhawk ADC4,14:Pixhawk ADC3,15:Pixhawk ADC6/Pixhawk2 ADC,103:Pixhawk SBUS
+    // @Values: -1:Not Used,8:CPN ADC,11:Pixracer,13:Pixhawk ADC4,14:Pixhawk ADC3,15:Pixhawk ADC6/Pixhawk2 ADC,103:Pixhawk SBUS
     AP_GROUPINFO("FL_PIN", 43, AP_BattMonitor_FuelLevel_Analog, _pin, -1),
 
  // index 44 unused and available
 
     // @Param: FL_FF
     // @DisplayName: First order term
     // @Description: First order polynomial fit term
-    // @Range: 0 10
+    // @Range: -10 10
```

Comment:
Please open an upstream for this. Don't add the "CPN ADC" addition, but these ranges are useful upstream.